LukeTHardy/backyard-buddy
# Lh mvp addplant

## Overview: 

- Created form for creating a new Plant, with various inputs for all the properties of a plant

## Testing: 

- Run debugger in API code
- Run `npm run dev` from client directory
- Login and navigate to /plants/newplant, or click Add Plant from the Plants page
- Verify the form. inputs, and save button all are rendered
